pubspec.yaml

```diff
@@ -1,14 +1,14 @@
 name: sms_autofill
 description: Flutter plugin to provide SMS code autofill support
-version: 1.3.1
+version: 2.0.0-nullsafety.1
```

Comment:
from what I understand putting `nullsafety ` is not needed anymore as in flutter 2.0 it's by default right ? We just need to set `environnement.flutter` to 2 or more to be sure it available only to nullsafety env.